python/kfserving/server.py

```diff
@@ -11,7 +11,7 @@
 DEFAULT_GRPC_PORT = 8081
 DEFAULT_PROTOCOL = "tensorflow/HTTP"
 
-parser = argparse.ArgumentParser()
+parser = argparse.ArgumentParser(add_help=False)
```

Comment:
Why did you change this?

Reply:
It was failing for me when I added parent as then both would by default add help methods.

Reply:
Odd. Should this be true then?

Reply:
I think it should be false as otherwise I get an error as both argsparse try to add the same `-h` argument.